File: pset1/mario/mario.c
#include<stdio.h>
#include <cs50.h>

int main (void)
{
    int height;
    do
    {
       height = get_int("Enter a positive number "); 
       printf(" %i\n", height);
       for(int i = 0; i < height;i++)
    { 
        for(int y = 0; y < height-i-1; y++)
        {
                printf(".");
        }
        for(int y = height-i-1; y < height; y++)
        {
                printf("#");
        }
        printf("\n");
    }
    printf("Stored: %i\n", height);
    }
    while(height > 0 && height < 8);
    
}

File: pset6/mario.py
from cs50 import get_int


len = get_int("Enter number: \n")
def d():
    print("#", end = '')
def a():
    print("#")
for i in range(len + 1):
    for j  in range(len + len - i):
        print(" " , end = '')
    for g in range(len - len + i):
        d()
    print(" ")
for i in range(len + 1):
    for j  in range(len + len - i):
        print(" " , end = '')
    for g in range(len - len + i):
        d()



